### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Marenz**
Mostly just a copy

### CODE REVIEW

**Review by leandro-lucarella-frequenz — COMMENTED** [review]
Sorry, but I would postpone this. Now the main entry point for this project will probably be the cookiecutter template, you probably won't need to do the manual install, so it doesn't make a lot of sense to have that in the readme (#39 updates the readme to show how to create a new repo using cookiecutter).

Also the next step will be to generate the documentation, so the readme can point to that for users that want a deeper dive. If you want to give visibility to the docs **right now** in the readme, I think it makes more sense to just say the project is under heavy development and put a link to https://github.com/frequenz-floss/frequenz-repo-config-python/blob/main/src/frequenz/repo/config/__init__.py if anyone wants to see the docs until they are properly generated.

File: README.md [hunk 1]
```diff
@@ -1,10 +1,59 @@
...
+- Libraries (lib)
 
-* Trivial build of `noxfile.py` with some predefined sessions with all common
-  checks.
-* Tools to build protobuf/grpc files as Python, including type information.
+## Installation
+
+The Frequenz project setup tools can be installed via pip:
+
+~~~bash
```
Comment (leandro-lucarella-frequenz) [inline README.md:14]: Just curious, why `~~~`? I didn't even knew it was a supported syntax.

Comment (shsms) [reply README.md:14]: Sometimes you forget to press shift,  then you get ~~~

Comment (Marenz) [reply README.md:14]: Mostly because that way chatGPT wouldn't render it but give it in a nice copy-able block :grin:

**Comment by Marenz** [comment]
Closing it for now as its becoming obsolete

**Pull request closed without merging**